roles/edpm_nova/tasks/configure.yml

```diff
@@ -101,6 +86,50 @@
   tags:
     - edpm_users
 
+- name: Sync nova config files
+  become: true
+  tags:
+    - configure
+    - nova
+  ansible.posix.synchronize:
+    src: "{{ edpm_nova_config_sync_src }}"
+    dest: "{{ edpm_nova_config_dest }}"
+    recursive: true
+    delete: true
+    mode: "push"
+    owner: true
```

Comment:
we do not want to preserve the owner of the files we copy.
they should be owned by the ansible user